index.html

```diff
@@ -155,9 +155,17 @@ <h3>Example</h3>
     "VerifiableCredential",
     "Iso18013DriversLicenseCredential"
   ],
-  "issuer": "did:key:z6MkjxvA4FNrQUhr8f7xhdQuP1VPzErkcnfxsRaU5oFgy2E5",
-  "issuanceDate": "2018-01-15T10:00:00.0000000-07:00",
-  "expirationDate": "2022-08-27T12:00:00.0000000-06:00",
+  "issuer": {
+    "id": "did:key:z6MkjxvA4FNrQUhr8f7xhdQuP1VPzErkcnfxsRaU5oFgy2E5",
+    "name": "Utopia Department of Motor Vehicles",
+    "url": "https://dmv.utopia.example/",
+    "image": "https://dmv.utopia.example/logo.png"
+  },
+  "issuanceDate": "2023-11-15T10:00:00.0000000-07:00",
+  "expirationDate": "2028-11-15T12:00:00.0000000-06:00",
+  "name": "Utopia Driver's License",
+  "image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUg...kSuQmCC",
+  "description": "Susan Turner's Utopia Driver's License.",
```

Comment:
This next remark might be nitpicking: technically "Susan Turner" is the credential subject and "Susan Turner's Utopia Driver's License" is what's inside the `driversLicense` term. Outside of the credential subject is not the driver's license but an assertion by the license issuer that the subjects possesses a license. So I would maybe change that description or move it inside `driversLicense`. But again that's just nitpicking.

Reply:
Yes, what you say is technically accurate. 

The digital wallets that we've been working with use the `name`, `description`, and `image` properties to render previews of the VC in the wallet. The `description` property on the VC is used for a more detailed explanation about the VC. I was trying to provide some text that would be useful in the wallets that use the description property to render this sort of information.